### PR TITLE
feat(aws): add inbound eventbridge connector template

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

Added inbound aws eventbridge connector template

## Related issues
tested with next fix : https://github.com/camunda/connectors-bundle/pull/913

depend on https://github.com/camunda/connectors-bundle/pull/920

Screenshot 2023-07-20 at 20 52 11
Screenshot 2023-07-20 at 20 52 24
Screenshot 2023-07-20 at 20 52 32
Screenshot 2023-07-20 at 20 52 38
Screenshot 2023-07-20 at 20 52 47

### CODE REVIEW

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.shouldValidateHmac"
+      }
+    },
+    {
+      "id": "authorizationType",
+      "label": "Authorization type",
+      "group": "authorization",
+      "description": "Choose the authorization type.",
```
Comment (igpetrov) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json:129]:
```suggestion
      "description": "Choose the authorization type",
```

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+          "value": "APIKEY"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.auth.type"
+      }
+    },
+    {
+      "label": "JWK url",
```
Comment (igpetrov) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json:156]:
```suggestion
      "label": "JWK URL",
```

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+      "description": "Well-known url of JWKs",
```
Comment (igpetrov) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json:157]:
```suggestion
      "description": "Well-known URL of JWKs",
```

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+      "condition": {
+        "property": "authorizationType",
+        "equals": "APIKEY"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "API Key locator",
```
Comment (igpetrov) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json:258]:
```suggestion
      "label": "API key locator",
```

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-start-event.json
Comment (igpetrov) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-start-event.json:1]: Same proposals as above

**Review by igpetrov — CHANGES_REQUESTED** [review]
LGTM, minor suggestions

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-start-event.json
Comment (igpetrov) [reply connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-start-event.json:1]: the `key` in label should also start from small `k`.

**Review by igpetrov — APPROVED** [review]

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+    "bpmn:IntermediateThrowEvent"
+  ],
+  "elementType": {
+    "value": "bpmn:IntermediateCatchEvent",
+    "eventDefinition": "bpmn:MessageEventDefinition"
+  },
+  "groups": [
+    {
+      "id": "endpoint",
+      "label": "Webhook configuration"
```
Comment (sbuettner) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json:23]: Lets call it `API destination` as this is the term AWS uses as well

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json [hunk 6]
```diff
@@ -0,0 +1,343 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Amazon EventBridge Connector",
+  "id": "io.camunda.connectors.AWSEventBridge.intermediate.v1",
+  "version": 1,
+  "description": "Send events to AWS EventBridge",
```
Comment (sbuettner) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-intermediate.json:6]: "Receive events from AWS EventBridge"

File: connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-start-event.json [hunk 7]
```diff
@@ -0,0 +1,303 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Amazon EventBridge Connector",
+  "id": "io.camunda.connectors.AWSEventBridge.startEvent.v1",
+  "version": 1,
+  "description": "Send events to AWS EventBridge",
```
Comment (sbuettner) [inline connectors/aws/aws-eventbridge/element-templates/aws-eventbridge-connector-start-event.json:6]: see above

**Review by sbuettner — CHANGES_REQUESTED** [review]

**Review by sbuettner — APPROVED** [review]

**Pull request merged**